beacon-chain/rpc/prysm/v1alpha1/validator/proposer_test.go

```diff
@@ -140,7 +135,7 @@ func TestProposer_ComputeStateRoot_OK(t *testing.T) {
 		StateGen:          stategen.New(db),
 	}
 	req := util.NewBeaconBlock()
-	req.Block.ProposerIndex = 21
+	req.Block.ProposerIndex = 84
 	req.Block.ParentRoot = parentRoot[:]
```

Comment:
Is this because minimal config now?

Reply:
Similar thing as in https://github.com/prysmaticlabs/prysm/pull/11268#discussion_r969548418. The calculated proposer index is different now.